src/test/java/com/github/bordertech/utils/SymbolsTest.java

```diff
@@ -57,4 +64,19 @@ public void testAsciifySymbolWithMarks() {
 		String out = "(C) (R) (TM)";
 		Assert.assertEquals("Translate mark symbols", out, Symbols.asciifySymbol(in));
 	}
+
+	@Test
+	public void testAsciifyAllTheWhiteSpace() {
+		String in = "\u0020 \u00A0 \u1680 \u180E \u2000 \u2001 \u2002 \u2003 \u2004 \u2005 \u2006 \u2007 \u2008 \u2009 \u200A \u200B \u202F \u205F \u3000 \uFEFF";
+		String out = "                                       ";
```

Comment:
Up to you but I'm wondering if this test might be easier to process visually if the `in` is an array and each character is compared to a single `" "`? No big deal, I'll leave it up to you, just food for thought I don't really mind.

Reply:
I will remove the in between blanks and I will split the space chars from the no-space chars, this should make it look better.